### PR TITLE
Added flake8.

### PR DESCRIPTION
**Pull request by hramezani**
I haven't change any part of the code and it is just a change in formatting.

### CODE REVIEW

**Comment by hramezani** [comment]
@slashmili  Do you have time to take a look at this?

File: t/test.py [hunk 1]
```diff
@@ -126,11 +126,6 @@ def test_subtract_time_delta(self):
         self.assertEqual(new_date.month, 4)
         self.assertEqual(new_date.day, 21)
         self.assertEqual(new_date.locale, 'nl_NL')
-    
-    def test_subtract_datetime_date(self):
```
Comment (farzadghanei) [inline t/test.py:130]: I'm curious why a formatting commit has removed this test case. could you please elaborate on this @hramezani?

**Comment by farzadghanei** [comment]
Thanks for the PR @hramezani , it definitely improves formatting of the project. :+1: Just had a question regarding a removed test case.

Personally I prefer the approach that [black formatter](https://pypi.org/project/black/) takes, to just enforce and fix formatting issues rather than showing warnings. But maybe we can discuss that further for the project and if @slashmili agrees, we can use that in a later PR.

File: t/test.py (hunk 1 above)
Comment (hramezani) [reply t/test.py:130]: It was a duplicate test

**Comment by hramezani** [comment]
@farzadghanei 

> Personally I prefer the approach that black formatter takes

Me too. Yeah, we can use it later.

**Review by farzadghanei — APPROVED** [review]

**Pull request merged**